sdk/src/test/java/com/stormpath/sdk/UserProfileTest.java

```diff
@@ -0,0 +1,119 @@
+package com.stormpath.sdk;
+
+import com.stormpath.sdk.models.UserProfile;
+
+import org.junit.Test;
+
+import java.net.HttpURLConnection;
+
+import okhttp3.mockwebserver.RecordedRequest;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Matchers.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.stub;
+import static org.mockito.Mockito.verify;
+
+public class UserProfileTest extends BaseTest {
```

Comment:
It's in https://github.com/edjiang/stormpath-framework-spec/blob/master/user-context.md